source/myscript.js

```diff
@@ -1,109 +1,164 @@
-$(document).ready(function(){
+$(document).ready(function () {
 	// Init GTM dataLayer for Analytics
 	var dataLayer = window.dataLayer || [];
+	var eventValue = 0;
+	var rating = '';
 
-	var thermometerHtml = 
-		"<div class='c-thermometer'> \
-			<div class='c-thermometer__trigger'> \
-				<div>🙂</div> \
+
+	var thermometerHtml =
+		" <div class='c-thermometer-popup'> \
+			<div class='c-thermometer-popup__close'>×</div> \
+			<svg width='19' height='19' viewBox='0 0 19 19' fill='none' xmlns='http://www.w3.org/2000/svg'><circle cx='9.5' cy='9.5' r='9.5' fill='#166DE0' fill-opacity='0.08'/><g clip-path='url(#clip0)'><path d='M15.0172 6.43306L7.81719 13.6331L4.51959 10.3211L5.36919 9.48586L7.81719 11.9339L14.1676 5.58346L15.0172 6.43306Z' fill='#166DE0'/></g><defs><clipPath id='clip0'><rect width='13.4118' height='11.1765' fill='white' transform='translate(2.79395 3.35303)'/></clipPath></defs></svg> \
+			<h5>Thank you!</h5> \
+			<p>We appreciate your feedback on this page.</p> \
+		</div> \
+		<div class='c-thermometer-modal__container'> \
+			<div> \
+				<div class='c-thermometer-modal__content'> \
+					<h3>Share your thoughts</h3> \
+					<p>Your feedback helps us improve. Let us know your thoughts about this page.</p> \
```

Comment:
```suggestion
					<p>Your feedback helps us improve. Share your thoughts with us here or join the Documentation channel on our Community server.</p> \
```

Reply:
Not sure if we can add hyperlinks here but if so, I'd like to link to the Documentation channel.